server/handlers/result_handlers.py

```diff
@@ -131,6 +130,19 @@ def ready(self):
 
         return self
 
+    def prepare_errors_payload(self, error_batch):
+        transformed_errors = []
+        for error in error_batch:
+            error = error.dict()
+            transformed_error = {
```

Comment:
can we conform with FailedPrediction model right away, avoiding redundant conversions?

Reply:
What we get in the result handler is a LSEBatchItem 
```
[LSEBatchItem(task_id=90040, output=None, error=True, message='ValidationError', details="1 validation error for Output\n  Invalid JSON: expected value at line 1 column 1 [type=json_invalid, input_value='// Sample implementation...esult === output;\\n};\\n', input_type=str]\n    For further information visit https://errors.pydantic.dev/2.8/v/json_invalid")]
```

Not sure we want to change that model, since it needs to work for both successful and failed predictions

Reply:
also a good idea, could be done here or in a later PR